cloudify_agent/resources/script/windows-download.ps1.template

```diff
@@ -3,6 +3,9 @@
 # All output should be UTF-8 (default is UTF-16).
 $PSDefaultParameterValues['*:Encoding'] = 'utf8'
 
+# Stop whenever a non-terminating cmdlet generates an error.
+$ErrorActionPreference = "Stop"
```

Comment:
Note that this is *not* equivalent to `bash -e`. It doesn't apply to running external programs (and `cfy-agent` is an external program).